dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/restmetrics/RestMetricsTest.java

```diff
@@ -120,41 +125,43 @@ private static String getHost() {
     public void testCoreMethods() throws IOException {
 
         // Execute a get method with varying numbers of parameters.
-        runGetMethod(0, 200, "/restmetrics/rest/restmetrics", "Metrics!");
+        runOptionsMethod(0, 200, "/restmetrics/rest/restmetrics", "Metrics!");
```

Comment:
I'm wondering if there isn't a better way to handle the "indexing"... maybe start with a counter at 0 and then increment?  It seems like this test will be difficult to maintain/update - especially after we haven't looked at it for a few months.

Reply:
agreed